docs/interactions/Receiving_and_Responding.md

```diff
@@ -32,26 +32,27 @@ For [Message Components](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/) it includes ide
+| name            | string                                                                                                                                                                                                     | the [`name`](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-structure) of the invoked command                                                 | Application Command, Application Command Autocomplete                                                                                              |
+| type            | integer                                                                                                                                                                                                    | the [`type`](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-structure) of the invoked command                                                 | Application Command, Application Command Autocomplete                                                                                              |
+| resolved?       | [resolved data](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-resolved-data-structure)                                                                                                    | converted users + roles + channels                                                                                                                                                    | Application Command                                                                                                                                |
+| options?        | array of [application command interaction data option](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-interaction-data-option-structure)                           | the params + values from the user                                                                                                                                                     | Application Command                                                                                                                                |
+| options?        | array of [application command autocomplete interaction data option](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-autocomplete-interaction-data-option-structure) | the params + values from the user                                                                                                                                                     | Application Command Autocomplete                                                                                                                   |
```

Comment:
In my opinion, I think it'd just be better to add the focused field to the `Application Command Interaction Data Option Structure` and mention it's only available for autocomplete interactions, like how options is mentioned for subcommands/subcommand groups, rather than having options in the table twice as that may (and probably would) confuse people

Reply:
You are probably right.
Will do later